statsmodels/base/model.py

```diff
@@ -1508,7 +1508,8 @@ def t_test(self, r_matrix, cov_p=None, scale=None, use_t=None):
                 not hasattr(self, 'cov_params_default')):
             raise ValueError('Need covariance of parameters for computing '
                              'T statistics')
-        if num_params != self.params.shape[0]:
+        params = self.params.ravel()
```

Comment:
Any chance of row/column order causing trouble?

Reply:
These models are unfortunately broken by design. params should always be a 1 d array even in multivariate models. MV models could expose some other property to show 2-d parameter arrays.

Reply:
vec is the correct operation to go from stack to 1d.